### PR TITLE
Backend: Add feature for challenge host to toggle the leaderboard sorting order

### PR DESCRIPTION
**Pull request by Sanji515**
**Issue:**

Challenge host unable to toggle the leaderboard sorting order.

**Fix:**

Added backend feature to toggle the leaderboard sorting order by challenge host

#### Changes proposed in this pull request:

- [x] Added new field `leaderboard_order_is_decreasing field` in `ChallengePhase` model.

- [x] Added this field for the admin and `ChallengePhaseCreateSerializer`.

- [x] Added a condition to toggle the sorting order a/c to this field in the leaderboard API.

- [x] Update the doc `configuration.md` for having `leaderboard_order_is_decreasing` field under `challenge_phases` of `challenge_config.yml` file

- [x] Updated the backend tests

### CODE REVIEW

File: apps/challenges/migrations/0055_added_leaderboard_order_is_descending_field.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (RishabhJain2018) [inline apps/challenges/migrations/0055_added_leaderboard_order_is_descending_field.py]: Please change the file name to - `add_leaderboard_order_is_descending_field_in_phase_model`

File: apps/challenges/admin.py [hunk 2]
```diff
@@ -71,8 +71,9 @@ class ChallengePhaseAdmin(ImportExportTimeStampedAdmin):
         "test_annotation",
         "is_public",
         "leaderboard_public",
+        "leaderboard_order_is_descending"
     )
-    list_filter = ("leaderboard_public", "start_date", "end_date")
+    list_filter = ("leaderboard_public", "leaderboard_order_is_descending", "start_date", "end_date")
```
Comment (RishabhJain2018) [inline apps/challenges/admin.py]: I don't think we need it here.

Comment (Sanji515) [reply apps/challenges/admin.py]: @RishabhJain2018 done

File: apps/challenges/migrations/0055_added_leaderboard_order_is_descending_field.py (hunk 1 above)
Comment (Sanji515) [reply apps/challenges/migrations/0055_added_leaderboard_order_is_descending_field.py]: @RishabhJain2018 done

**Comment by Sanji515** [comment]
Closing this, will open new PR for this feature.

**Pull request closed without merging**